lib/rubocop/result_cache.rb

```diff
@@ -128,7 +128,8 @@ def rubocop_checksum
         begin
           lib_root = File.join(File.dirname(__FILE__), '..')
           bin_root = File.join(lib_root, '..', 'bin')
-          source = Find.find(lib_root, bin_root).sort.map do |path|
+          source_files = $LOADED_FEATURES + Find.find(lib_root, bin_root).to_a
```

Comment:
I could record the things we `require`d but this is safer. I didn't profile it, but if this turns out to be the lowest hanging startup fruit we can change it.